### PR TITLE
Frontend: Update react-router to v6

### PR DESCRIPTION
**Pull request by petertrr**
### What's done:
* Bump kotlin wrappers to pre.268
* Change API usages of react-router: https://reactrouter.com/docs/en/v6/upgrading/v5

### CODE REVIEW

File: save-frontend/src/main/kotlin/org/cqfn/save/frontend/App.kt
Comment (petertrr) [inline save-frontend/src/main/kotlin/org/cqfn/save/frontend/App.kt:156]: react-router v6 has dropped support for regexes in patterns, but trailing `*` now indicates that the entire ending of path should be captured. I tested it manually and it works as expected.

**Comment by akuleshov7** [comment]
What is the need of this update?

**Comment by petertrr** [comment]
> What is the need of this update?

kotlin-wrappers have updated react-router and it was blocking us from updates. And since kotlin-wrappers release only for current version of Kotlin and generally change quite fast, it's worth for us to update too

**Review by kgevorkyan — APPROVED** [review]
but let's also wait for @akuleshov7 for final decision, I remember, he had some doubts

**Review by akuleshov7 — CHANGES_REQUESTED** [review]
don't merge this week

**Review by akuleshov7 — APPROVED** [review]

**Pull request merged**
